InterviewFlashCards/AnswersViewController.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  AnswersViewController.swift
+//  InterviewFlashCards
+//
+//  Created by Charles Kang on 1/5/17.
+//  Copyright © 2017 Varindra Hart. All rights reserved.
+//
+
+import UIKit
+
+class AnswersViewController: UITableViewController {
+    
+    var flashCard = IFCFlashCard()
+    
+    @IBOutlet weak var answerLabel: UILabel!
+    @IBOutlet weak var answerImageView: UIImageView!
+    @IBOutlet weak var backButton: UIButton!
+    @IBOutlet weak var nextButton: UIButton!
+    @IBOutlet weak var answer: UILabel!
+    @IBOutlet weak var paginationLabel: UILabel!
+    var tempAnswerImageView: UIImageView!
+    var tempAnswerLabel: UILabel!
+    var isTapped = false
+    var index = 0
+    var leftGesture: UISwipeGestureRecognizer!
+    var rightGesture: UISwipeGestureRecognizer!
+    var tapGesture: UITapGestureRecognizer!
```

Comment:
Force unwrapped variables are a possible source of crashes, I would recommend against this.